lib/ukrust/Makefile.uk

```diff
@@ -0,0 +1,67 @@
+$(eval $(call addlib_s,libukrust,$(CONFIG_LIBUKRUST)))
+
+CINCLUDES-$(CONFIG_LIBUKRUST) += -I$(LIBUKRUST_BASE)/ukrust_sys
+
+RUSTCFLAGS-$(CONFIG_LIBUKRUST) += --extern ukrust_sys -L $(LIBUKRUST_BUILD)/ --extern alloc \
+		--target $(LIBUKRUST_BASE)/target.json \
+		-Zallow-features=allocator_api,bench_black_box,concat_idents,global_asm,try_reserve
+
+
+export RUST_BINDINGS_FILE=$(LIBUKRUST_BUILD)/bindings_generated.rs
```

Comment:
`export LIBUKRUST_BINDINGS=<...>`? ... see my next comment why I am suggesting this.